dashboard/app/views/levels/_content.html.haml

```diff
@@ -9,10 +9,12 @@
   / content levels.
   %script{src: webpack_asset_path('js/blockly.js')}
   %script{src: webpack_asset_path("js/#{js_locale}/blockly_locale.js")}
-  / TODO [FND-972] Only the associated level type's blocks.
-  %script{src: webpack_asset_path("js/#{js_locale}/turtle_locale.js")}
+  - associated_blocks_json = {}
+  - if level.try(:associated_blocks) && level.associated_blocks
+    %script{src: webpack_asset_path("js/#{js_locale}/#{level.associated_blocks}_locale.js")}
+    - associated_blocks_json = level.associated_blocks.to_json
 
-  %script{src: webpack_asset_path('js/levels/_content.js')}
+  %script{src: webpack_asset_path('js/levels/_content.js'), data: {associatedblocks: associated_blocks_json}}
```

Comment:
do we need this data here as well as above?

Reply:
I'm not sure I follow? We need to pass down the associated block type and I don't see where else we send this data to the frontend?

Reply:
oh, I think I had misread what was going on in the lines above, and thought we were passing this data to the frontend in multiple places. Reading it again, this looks how I'd expect!